mapswipe_workers/mapswipe_workers/project_types/base/project.py

```diff
@@ -587,14 +593,20 @@ def create_tasks_txt_file(self, groupsOfTasks):
                 # these common attributes don't need to be written
                 # to the project_type_specifics since they are
                 # already stored in separate columns
-                common_attributes = ["projectId", "groupId", "taskId", "geometry"]
+                common_attributes = [
+                    "projectId",
+                    "groupId",
+                    "taskId",
+                    "geometry",
+                    "geojson",
```

Comment:
perfect, that put this here. I think so far we stored the geometry twice. So this is much better.